src/conventions/h-group/clue-interpretation/connecting-cards.js

```diff
@@ -159,15 +160,75 @@ function find_unknown_connecting(game, giver, target, reacting, identity, connec
 		}
 		// Finessed card is delayed playable
 		else if (game.level >= LEVEL.INTERMEDIATE_FINESSES && state.play_stacks[finesse.suitIndex] + 1 === finesse.rank) {
-			// Could be duplicated in giver's hand - disallow hidden finesse
-			if (state.hands[giver].some(c => c.clued && game.players[giver].thoughts[c.order].inferred.has(identity))) {
+			let bluff = false;
```

Comment:
This can be simplified into one statement without needing to use `let`:
```js
const bluff = game.level >= LEVEL.BLUFFS &&
    connected.length == 1 &&
    ((giver + 1) % state.numPlayers) == reacting
    && !common.thinksLoaded(state, reacting);
```
This pattern also appears on line 347 of own-finesses.js.